apps/api/src/app/logs/logs.controller.ts

```diff
@@ -1,4 +1,8 @@
 import { Controller } from '@nestjs/common';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { ApiKeyAuth } from '../shared/framework/swagger/api.key.security';
 
 @Controller('/logs')
+@ApiBearerAuth()
+@ApiKeyAuth()
 export class LogsController {}
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** this can be removed

Reply:
@rifont same here